stable/mariadb/templates/deployment.yaml

```diff
@@ -63,7 +71,7 @@ spec:
           mountPath: /bitnami/mariadb/conf/my_custom.cnf
           subPath: my.cnf
         - name: data
-          mountPath: /bitnami/mariadb
+          mountPath: /bitnami
```

Comment:
Changing the mountPath would break chart upgrades? If we revert this change will it be possible to perform clean chart upgrades?

Reply:
You're right! For the upgrade to work, the mount path has to be `/bitnami/mariadb` for the container and the initContainer.